btrack/models.py

```diff
@@ -293,10 +287,11 @@ class HypothesisModel:
     relax: bool
     name: str = "Default"
 
-    @staticmethod
-    def load(filename: os.PathLike):
-        """Load a model from file."""
-        return utils.read_hypotheis_model(filename)
+    @validator("hypotheses", pre=True)
+    def parse_hypotheses(cls, hypotheses):
+        if not all([h in H_TYPES for h in hypotheses]):
```

Comment:
I suspect this will work as a generator, i.e. without the `[]`
```suggestion
        if not all(h in H_TYPES for h in hypotheses):
```